packages/blocks/src/components/SplitButton/SplitButton.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { AutoImportIcon, XIcon } from '@mergestat/icons';
+
+type SplitButtonProps = {
+  children?: React.ReactNode;
+  onClickClose?: () => void;
+  onClick?: () => void;
+};
+
+export const SplitButton: React.FC<SplitButtonProps> = ({
+  children,
+  onClickClose,
+  onClick,
+}) => {
+  return (
+    <div className="t-flex-split t-split-gray text-base">
+      <label onClick={onClick}>
+        <AutoImportIcon className="t-icon" />
+        <button>{children || 'Create Auto Import'}</button>
+      </label>
+      <div className="t-divider" />
+      <XIcon className="t-icon" onClick={onClickClose} />
+    </div>
+  );
+};
```

Comment:
Heads up, the spacing here is a bit off around the icons:

<img width="244" alt="image" src="https://user-images.githubusercontent.com/57259/163288918-dd7a4cb1-9e89-411c-a2dd-1b8bf8056ee0.png">